source/conf.py

```diff
@@ -366,41 +96,38 @@
     ),
 ]
 
-# Documents to append as an appendix to all manuals.
-#
-# texinfo_appendices = []
+# -- Options for the linkcheck builder -------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-the-linkcheck-builder
 
-# If false, no module index is generated.
-#
-# texinfo_domain_indices = True
+linkcheck_ignore = [
+    "http://localhost:\\d+",
+    "https://test.pypi.org/project/example-package-YOUR-USERNAME-HERE",
+    "https://pypi.org/manage/*",
+    "https://test.pypi.org/manage/*",
+]
 
-# How to display URL addresses: 'footnote', 'no', or 'inline'.
-#
-# texinfo_show_urls = 'footnote'
+# -- Options for extlinks ----------------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/extensions/extlinks.html#configuration
 
-# If true, do not generate a @detailmenu in the "Top" node's menu.
-#
-# texinfo_no_detailmenu = False
+github_url = "https://github.com"
+github_repo_org = "pypa"
+github_repo_name = "packaging.python.org"
+github_repo_slug = f"{github_repo_org}/{github_repo_name}"
+github_repo_url = f"{github_url}/{github_repo_slug}"
+github_repo_issues_url = f"{github_url}/{github_repo_slug}/issues"
+github_sponsors_url = f"{github_url}/sponsors"
 
-# -- Options for extlinks extension ---------------------------------------
 extlinks = {
-    "issue": (f"{github_repo_issues_url}/%s", "#%s"),  # noqa: WPS323
-    "pr": (f"{github_repo_url}/pull/%s", "PR #%s"),  # noqa: WPS323
-    "commit": (f"{github_repo_url}/commit/%s", "%s"),  # noqa: WPS323
-    "gh": (f"{github_url}/%s", "GitHub: %s"),  # noqa: WPS323
-    "user": (f"{github_sponsors_url}/%s", "@%s"),  # noqa: WPS323
+    "issue": (f"{github_repo_issues_url}/%s", "#%s"),
+    "pr": (f"{github_repo_url}/pull/%s", "PR #%s"),
+    "commit": (f"{github_repo_url}/commit/%s", "%s"),
+    "gh": (f"{github_url}/%s", "GitHub: %s"),
+    "user": (f"{github_sponsors_url}/%s", "@%s"),
```

Comment:
```suggestion
    "user": (f"{github_sponsors_url}/%s", "@%s"),
    "pypi": ("https://pypi.org/project/%s", "%s"),
```

I think this could be real helpful.

Reply:
Can be handled separately, and with actual usage. Just adding the alias without using it seems a bit strange.

Issue created: https://github.com/pypa/packaging.python.org/issues/1370